### PR TITLE
Update dependent up to date

### PR DESCRIPTION
**Pull request by kookyleo**

### CODE REVIEW

File: lib/src/plugin/heat_map_tiles_provider.dart [hunk 1]
```diff
@@ -63,8 +65,10 @@ class HeatMapTilesProvider extends TileProvider {
 
     var localMin = 0.0;
     var localMax = 0.0;
-    CustomPoint tileOffset =
-        CustomPoint(options.tileSize * coords.x, options.tileSize * coords.y);
+    // CustomPoint tileOffset =
+    // CustomPoint(options.tileSize * coords.x, options.tileSize * coords.y);
```
Comment (leewrigg) [inline lib/src/plugin/heat_map_tiles_provider.dart]: I would suggest removing the old code entirely so it doesn't clog up the codebase as it won't be used again

```suggestion
```

File: lib/src/plugin/heatmap_data_provider.dart [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import 'dart:math' as math;
+import 'package:flutter_map/flutter_map.dart';
+
 import 'latlong.dart';
 
-import 'package:flutter_map/plugin_api.dart';
+// import 'package:flutter_map/plugin_api.dart';
```
Comment (leewrigg) [inline lib/src/plugin/heatmap_data_provider.dart]: As above

```suggestion
```

File: pubspec.yaml [hunk 3]
```diff
@@ -4,13 +4,30 @@ repository: https://github.com/tprebs/fluttermap_heatmap
 version: 0.0.5
 
 environment:
-  sdk: ">=2.17.0 <4.0.0"
+  # sdk: ">=2.17.0 <4.0.0"
```
Comment (leewrigg) [inline pubspec.yaml]: As above 

```suggestion
```

File: pubspec.yaml [hunk 4]
```diff
@@ -4,13 +4,30 @@ repository: https://github.com/tprebs/fluttermap_heatmap
...
 
 dependencies:
   flutter:
     sdk: flutter
-  flutter_map: ^4.0.0
-  latlong2: ^0.8.1
+  # flutter_map: ^4.0.0
+  flutter_map: ^6.0.0
+  # latlong2: ^0.8.1
+  latlong2: ^0.9.0
```
Comment (leewrigg) [inline pubspec.yaml]: As above

```suggestion
  flutter_map: ^6.0.0
  latlong2: ^0.9.0
```

File: pubspec.yaml [hunk 5]
```diff
@@ -4,13 +4,30 @@ repository: https://github.com/tprebs/fluttermap_heatmap
...
+#   sdk: '>=3.1.0 <4.0.0'
+
+# dependencies:
+#   flutter:
+#     sdk: flutter
+#   flutter_map: ^6.0.0
+#   latlong2: ^0.9.0
+
+# dev_dependencies:
+#   flutter_lints: ^2.0.3
```
Comment (leewrigg) [inline pubspec.yaml]: As above

```suggestion
  flutter_lints: ^2.0.3
```

**Review by leewrigg — COMMENTED** [review]
Great work @kookyleo 

I've left a few suggestions relating to code styling (removing old code rather than commenting it out) but aside from that, these changes look good to me 👍 

I'm intending to test the changes in the next day or 2; I'll drop in and approve once I've done this!

**Comment by tprebs** [comment]
Thanks @kookyleo and thanks @leewrigg  for reviewing this. I have been travelling the last couple of weeks so haven't had a chance to look at this. @kookyleo, can you update the PR with the changes suggested by lee? If not I will cherrypick your changes into a new branch and clean them up later this week

**Review by leewrigg — APPROVED** [review]
Apologies for taking longer than expected to test this. 
I've been using this branch for a few days and all seems to be working perfectly. Thanks @kookyleo!

**Comment by kookyleo** [comment]
I apologize for the delay, I have been busy with other matters recently.

**Review by tprebs — APPROVED** [review]

**Comment by tprebs** [comment]
Thanks @kookyleo and @leewrigg . Merged and new release 0.0.6 available

**Comment by leewrigg** [comment]
Great, thanks @tprebs

**Pull request merged**